testsuite/api/connections.py

```diff
@@ -90,8 +90,7 @@ def printStatus(status, expected):
 ## status:  [correct] error
 ## error:   [addConnection] Type mismatch in connection: sc1.u2 -> sc2.y2
 ## status:  [correct] error
-## error:   [addConnection] Connector is already connected: sc1.u1
-## status:  [correct] error
+## status:  [wrong] ok
```

Comment:
@arun3688 and this as well - but should be the same issue as the one above.

Reply:
@lochel this should be fixed with this https://github.com/OpenModelica/OMSimulator/pull/925/commits/d646dbc5ac9c9c687298960bcdd2617225d7d9e6